app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/DeveloperOptionsView.java

```diff
@@ -219,9 +244,33 @@ private void setLocalAddon(boolean value, boolean doApply) {
         }
     }
 
+    private void setUseWolvicUA(boolean value, boolean doApply) {
+        mBinding.wolvicUserAgentSwitch.setOnCheckedChangeListener(null);
+        mBinding.wolvicUserAgentSwitch.setValue(value, false);
+        mBinding.wolvicUserAgentSwitch.setOnCheckedChangeListener(mUseWolvicUAListener);
+
+        if (doApply) {
+            SettingsStore.getInstance(getContext()).setUseWolvicUA(value);
+        }
+    }
+
+    private void setUaMode(int checkId, boolean doApply) {
+        mBinding.uaRadio.setOnCheckedChangeListener(null);
+        mBinding.uaRadio.setChecked(checkId, doApply);
+        mBinding.uaRadio.setOnCheckedChangeListener(mUaModeListener);
+
+        SettingsStore.getInstance(getContext()).setUaMode((Integer) mBinding.uaRadio.getValueForId(checkId));
+    }
+
+    @Override
+    public Point getDimensions() {
+        return new Point(WidgetPlacement.dpDimension(getContext(), R.dimen.settings_dialog_width),
+                WidgetPlacement.dpDimension(getContext(), R.dimen.developer_options_height));
+    }
```

Comment:
I don't see the relationship of this with anything else. 

Is it something that was missing? Does it fix any particular issue? If that's the case it should go in a different PR.

Reply:
This makes the dialog a bit taller, which is needed after adding the two settings for the User-Agent.

The alternative was to keep the dialog with the same height, but that looks worse:

![08b9cbb2b6a8529c27325e8a27f43551](https://user-images.githubusercontent.com/1412060/214752063-322dcf90-30ea-4e2f-939a-df2d747ad585.png)